students/ScottL/session05/Mailroom_Part3.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+
+import sys
+
+global_donation_list = {"Tom Petty": [5000, 15000],
+                        "Neil Diamond": [20000, 30000],
+                        "Elton John": [25000, 25000, 40000],
+                        "John Lennon": [25000],
+                        "Mick Jagger": [20000, 10000]}
+MENU_HEADER_1 = "\nMenu Options:"
+MENU_HEADER_2 = "\n" + "-" * len(MENU_HEADER_1)
+MENU_SELECTIONS = {1: "Send a 'Thank You' note to individual donor",
+                   2: "Display a report",
+                   3: "Send 'Thank You' letters to all donors",
+                   4: "Quit"}
+REPORT_HEADER_1 = "\n{:<26}{:<12}{:^17}{:>11}".format(
+                    "Donor Name", "Total Dollars", "Number", "Avg Amount")
+REPORT_HEADER_2 = "\n" + "-" * len(REPORT_HEADER_1)
+DONOR_LIST_HEADER = "\nDonor Name:\n" + "-" * 20
+
+
+def display_user_menu():
+    """Display the contents of the menu options for the user."""
+    print(MENU_HEADER_1, MENU_HEADER_2)
+    for item in MENU_SELECTIONS:
+        print(item, ": ", MENU_SELECTIONS[item])
```

Comment:
here is a good spot:
`print(item, ": ", MENU_SELECTIONS[item]) for item in MENU_SELECTIONS`

Reply:
I'm not sure why in the text it says don't use them for print.  My only thought is that typically in comps they are used cause they are faster than a for loop.  But nothing in any of these programs really require speed, it more about practice and recognizing when they can be used.